tools/cli/NearObjectCliHandler.cxx

```diff
@@ -19,8 +21,12 @@ NearObjectCliHandler::HandleStartRanging(std::shared_ptr<uwb::UwbDevice> uwbDevi
 {
     auto callbacks = std::make_shared<nearobject::cli::NearObjectCliUwbSessionEventCallbacks>();
     auto session = uwbDevice->CreateSession(callbacks);
-    session->Configure(sessionData);
-    session->StartRanging();
+    if (not sessionData.uwbConfigurationAvailable) {
+        PLOG_WARNING << "did not start ranging due to missing configuration";

```

Comment:
I don't think this is correct. If the user of the nocli tool is starting a ranging session, that means they are starting a session from scrach and will be setting the config params as CLI args. So, they will see this warning each time and the session will never be configured or ran, which doesn't make sense. Unless I'm misunderstanding the if statement here

Reply:
My understanding is that HandleStartRanging is called when the user enters the range start command via CLI. This means they need to input the config params to start ranging, which I think is correct.

Reply:
The 
```
nocli uwb range start [--options]
```
command has the user enter in the config params via the --options

Reply:
Actually you raised a good point. Because UwbConfiguration right now is a bunch of std::optional this means we don't actually need a boolean to keep track of whether there is a UwbConfiguration. And it looks like that boolean is really being used anywhere either. So I'll remove this unnecessary check.